src/main/java/models/AccountModel.java

```diff
@@ -0,0 +1,32 @@
+package models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Random;
+
+@Getter
+@Setter
+public class AccountModel {
+    public CustomerModel customerModel;
+    public Boolean isSubscribedToNewsletter;
+    public String loginName;
```

Comment:
Why we need that as this is already part of customermodel? Maybe we should just move those here?

Reply:
login and password were deleted from CustomerModel and now they are part only of AccountModel